jax/_src/custom_derivatives.py

```diff
@@ -749,30 +791,33 @@ def _custom_vjp_call_jaxpr_abstract_eval(*_, fun_jaxpr, **__):
 def _custom_vjp_call_jaxpr_jvp(
     primals, tangents, *, fun_jaxpr: core.ClosedJaxpr,
     fwd_jaxpr_thunk: Callable[[], Tuple[core.Jaxpr, Sequence[Any]]],
```

Comment:
```suggestion
    fwd_jaxpr_thunk: Callable[[???], Tuple[core.Jaxpr, Sequence[Any]]],
```

Reply:
I think maybe `Callable[..., Tuple[core.Jaxpr, Sequence[Any]]]` is the best annotation.